tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/driftingcontractor.ts

```diff
@@ -0,0 +1,21 @@
+import { multiline } from "common/string";
+import { Antagonist, Category } from "../base";
+
+export const CONTRACTOR_MECHANICAL_DESCRIPTION
+  = multiline`
+      Float onto the station and complete as many
+      contracts for your employer as you can!
+    `;
+
+const DriftingContractor: Antagonist = {
+  key: "driftingcontractor",
+  name: "Drifting Contractor",
+  description: [
+    multiline`A Syndicate agent that can spawn near the station, equipped with
+    top-of-the-line contractor gear, to complete contracts for the Syndicate.`,
+    CONTRACTOR_MECHANICAL_DESCRIPTION,
+  ],
```

Comment:
`CONTRACTOR_MECHANICAL_DESCRIPTION` isn't actually being imported by anything, so there's no need to have it as a separate `export` block.
```suggestion
import { Antagonist, Category } from "../base";

const DriftingContractor: Antagonist = {
  key: "driftingcontractor",
  name: "Drifting Contractor",
  description: [
    multiline`A Syndicate agent that can spawn near the station, equipped with
    top-of-the-line contractor gear, to complete contracts for the Syndicate.`,
    multiline`Float onto the station and complete as many
    contracts for your employer as you can!`,
  ],
```